src/components/site-header/AvatarDropdown.jsx

```diff
@@ -0,0 +1,41 @@
+import React, { useContext } from 'react';
+import PropTypes from 'prop-types';
+import { NavLink } from 'react-router-dom';
+import { getConfig } from '@edx/frontend-platform/config';
+import { AppContext } from '@edx/frontend-platform/react';
+import { AvatarButton, Dropdown } from '@edx/paragon';
+
+const AvatarDropdown = ({ showLabel }) => {
+  const { BASE_URL, LMS_BASE_URL, LOGOUT_URL } = getConfig();
+  const { enterpriseConfig, authenticatedUser } = useContext(AppContext);
+  const { username, profileImage } = authenticatedUser;
+  const dashboardLink = `/${enterpriseConfig.slug}`;
+  return (
+    <Dropdown>
+      <Dropdown.Toggle showLabel={showLabel} as={AvatarButton} src={profileImage.imageUrlMedium}>
+        {username}
+      </Dropdown.Toggle>
+      <Dropdown.Menu alignRight>
+        <Dropdown.Header className="text-uppercase">Switch Dashboard</Dropdown.Header>
+        <Dropdown.Item href={`${LMS_BASE_URL}/dashboard`}>Personal</Dropdown.Item>
+        <Dropdown.Item as={NavLink} to={dashboardLink}>Enterprise</Dropdown.Item>
+        <Dropdown.Divider className="border-light" />
+        <Dropdown.Item href={`${LMS_BASE_URL}/u/${authenticatedUser.username}`}>My Profile</Dropdown.Item>
+        <Dropdown.Item href={`${LMS_BASE_URL}/account/settings`}>Account Settings</Dropdown.Item>
+        <Dropdown.Item href="https://support.edx.org/hc/en-us">Help</Dropdown.Item>
```

Comment:
this URL is hardcoded to point to the edX support page on prod, should it be dynamic like the other URLs here?

Reply:
There isn't a stage version of the support page that I'm aware of so it's the same for both stage/prod. Good question, though!